features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1556,6 +1556,9 @@
             -->
             <ComplexMultiValuedAttributeSupportEnabled>{{scim.enable_complex_multivalued_attribute_support}}</ComplexMultiValuedAttributeSupportEnabled>
         </SCIMAuthenticators>
+
+        <!-- Restrict federated users from accessing the endpoint -->
+        <RestrictFederatedUserAccess>{{scim.restrict_federated_user_access}}</RestrictFederatedUserAccess>
```

Comment:
we should add default.json value as well since new infer.json process required default.json values. Shall we have the default value as true ? What was the final decision we made during the discussion?